nixos/tests/grafana-alloy.nix

```
@@ -0,0 +1,32 @@
+import ./make-test-python.nix ({ lib, pkgs, ... }:
+
+  let
+    nodes = {
+      machine = {
+        services.grafana-alloy = {
+          enable = true;
+        };
+        environment.etc."alloy/config.alloy".text = "";
+      };
+    };
+  in
+  {
+    name = "grafana-alloy";
+
+    meta = with lib.maintainers; {
+      maintainers = [ flokli ];
```

Happy to help maintain this one too, btw :)

```suggestion
      maintainers = [ flokli hbjydev ];
```